dts/bindings/ethernet/xlnx,axi-ethernet-base.yaml

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2024 CISPA Helmholtz Center for Information Security
+# SPDX-License-Identifier: Apache-2.0
+
+description: Xilinx AXI 1G/2.5G Ethernet Subsystem base bindings.
+
+include: [base.yaml]
+
+# multiple "compatible" properties match the same driver and options
+
+properties:
+  interrupts:
+    type: array
+    description: Xilinx AXI Ethernet has an optional interrupt for updating statistics.
+  interrupt-parent:
+    type: phandle
+    description: Interrupt controller that the DMA is connected to.
+  reg:
+    type: array
+    description: |
+      Control registers of the MDIO device are also used for the Ethernet MAC controller.
+      Thus, both devices should have a parent node with compatible "xlnx,axi-ethernet-subsystem-7.2"
+      which defines the reg property, and MAC and MDIO should NOT have a reg property.
+  clocks:
+    type: phandle-array
+  clock-frequency:
+    type: int
+  local-mac-address:
+    type: uint8-array
+    description: Ethernet adapter MAC address.
+    required: true
+
+  axistream-connected:
+    type: phandle
+    required: true
+    description: Xilinx AXI DMA that is connected to the data streams.
+
+  axistream-control-connected:
```

Comment:
this is not defined in Linux dt binding and I would like to have this as close as possible to Linux one. 
Are you able to setup design that you have data and control streams pointing to different location?

Reply:
You can disable the control stream altogether; it is only used for checksum offloading and a few other auxiliary features. You can also theoretically connect this to a custom IP (it is a different physical interface than the one used for data), but I am not aware of any use case for this.